source/features/clean-conversation-filters.tsx

```diff
@@ -55,7 +55,7 @@ async function hasProjects(): Promise<boolean> {
 
 async function hideProjects(): Promise<void> {
 	if (!await hasProjects()) {
-		(await elementReady('[data-hotkey="p"]'))!.parentElement!.remove();
+		(await elementReady('[data-hotkey="p"]'))?.parentElement!.remove();
```

Comment:
Line 57 exactly checks for projects. If we get to line 58, there should not be any projects.

Reply:
hmm I will double check what's going on

Reply:
> Line 57 exactly checks for projects.

Correct but it does not check if the projects dropdown exists.
I have the error on https://github.com/download-directory/download-directory.github.io/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Abug

Reply:
Of course it doesn't exist, it doesn't have any projects.

Reply:
That should already be covered:

https://github.com/sindresorhus/refined-github/blob/7c5c2a08129bb2941e1121c457b908e58f3a0d43/source/features/clean-conversation-filters.tsx#L48-L50

Reply:
Let's merge `hideProjects` and `hasProjects` into something that makes more sense. Currently the condition is "has projects" while it tries to be "should it hide the section"

Why are we even checking for projects if there's no sidebar section to hide at all?